l10n_be_cooperator_national_number/models/company.py

```diff
@@ -3,10 +3,28 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 
-from odoo import fields, models
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
 
 
 class ResCompany(models.Model):
     _inherit = "res.company"
 
+    display_national_number = fields.Boolean(string="Display National Number")
     require_national_number = fields.Boolean(string="Require National Number")
+
+    @api.constrains("display_national_number", "require_national_number")
+    def _check_national_number(self):
+        for company in self:
+            if company.require_national_number and not company.display_national_number:
+                raise ValidationError(
+                    _(
+                        "If the 'Require National Number' toggle is enabled,"
+                        " then so must the 'Display National Number' toggle."
```

Comment:
```suggestion
                        'If the "Require National Number" toggle is enabled,'
                        ' then so must the "Display National Number" toggle.'
```

i think that delimiting strings with double quotes is more common in user-facing text.